exporter/prometheusremotewriteexporter/README.md

```diff
@@ -36,6 +36,9 @@ The following settings can be optionally configured:
 - `headers`: additional headers attached to each HTTP request. 
   - *Note the following headers cannot be changed: `Content-Encoding`, `Content-Type`, `X-Prometheus-Remote-Write-Version`, and `User-Agent`.*
 - `namespace`: prefix attached to each exported metric name.
+- `queue_config`: fine tuning for the outgoing request queues.
+  - `size`: number of OTLP metrics that can be queued.
```

Comment:
I don't see this in the PRW parameters?  Just to confirm, we are introducing something new here, right?  Is there a way for us to stick with the existing PRW configs?

Reply:
This is new, PRW has aa setting for capacity but it represents the number of samples.